packages/node/types/bugsnag.d.ts

```diff
@@ -10,7 +10,7 @@ declare module "@bugsnag/core" {
       onUnhandledRejection?: (err: any, report: Bugsnag.Report, logger: Bugsnag.ILogger) => void;
       proxy?: string;
       projectRoot?: string;
-      terminateOnUnhandledRejection?: boolean;
+      sendCode?: string;
```

Comment:
Shouldn't `sendCode` be a boolean?

https://github.com/bugsnag/bugsnag-js/blob/2863a379a6bdbffdbd857a2c9f534a07d49d1de3/packages/plugin-node-surrounding-code/surrounding-code.js#L34-L38

Reply:
Thanks JM :)